contributors/design-proposals/auth/svcacct-token-volume-source.md

```diff
@@ -0,0 +1,138 @@
+# Service Account Token Volumes
+
+Authors:
+  @smarterclayton
+  @liggitt
+  @mikedanese
+
+## Summary
+
+Kubernetes is able to provide pods with unique identity tokens that can prove
+the caller is a particular pod to a Kubernetes API server. These tokens are
+injected into pods as secrets. This proposal proposes a new mechanism of
+distribution with support for [improved service account
+tokens](https://github.com/kubernetes/community/pull/1460) and explores how to
```

Comment:
A PR is not sufficient here.  We should make sure that anything referenced by a design doc is checked in and agreed upon.  We *have* to get out of the habit of not closing these things out.  We need to be able to actually make decisions.

Reply:
I can prioritize closing that out. The outstanding question is here:

https://github.com/kubernetes/community/pull/1460#discussion_r166636787

Reply:
If you want to use the KEP process (not required but might be nice -- hint hint) there is an idea of checking in stuff that is WIP and isolating the nut of disagreement to dedicated PRs.  That helps to find the areas agreement and get them locked down incrementally.